docs/managing-templates.md

```diff
@@ -134,3 +134,154 @@ Now let's make use of our new template.
         }
     ]
     ```
+
+## Adding a required field
+
+In may cases, we want users of the template to provide more detail, such as
+data that can be used to tailor the notifications based on the target at
+hand. This is accomplished via a [form fields](declarative-templates.md#example-form-fields).
+
+Let's make a new template that enables notifications via [Azure Devops Work
+Items](notifications/ado.md), where we require the user to specify the Area
+Path that the work items OneFuzz will create.
+
+This example will demonstrate setting the following:
+* `Project` via the project name specified during job creation.
+* `Area Path` via the new required field.
+* `Iteration Path` via a new optional field, with a default value in the template.
+
+1. Enable support for declarative templates.
```

Comment:
This might be a good place to call out:
```suggestion
1. Enable support for declarative templates. This is required because declarative templates are not yet stabilized.
```
Might be worth emphasizing since `--enable_feature` is basically not something a user otherwise has any reason to be aware of, and this will probably be the first time they see it.